modules/cluster-logging-deploying-about.adoc

```diff
@@ -161,14 +161,16 @@ spec:
   logStore:
     type: "elasticsearch"
     elasticsearch:
-      nodeCount: 2
+      nodeCount: 3
       resources:
         limits:
-          memory: 2Gi
+          memory: 16Gi
         requests:
           cpu: 200m
-          memory: 2Gi
-      storage: {}
+          memory: 16Gi
+        storage:
+          storageClassName: "gp2"
```

Comment:
To align with another PR, shall we replace gp2?

Reply:
This CR is supposed to show what a modified CR looks like. I think having a specific storageClassName is OK here.